### DISCUSSION TITLE
it seems Vault QA doesn't work?

### DISCUSSION THREAD

**Discussion by breakTBB** [logancyang/obsidian-copilot#559] [Q&A]
I tried to ask a question based on a note in my vault, but Model didn't know anything about my question

**Comment by KyleJFischer** [comment]
What additional information can you provide? Is your network connection setup? Launch the console and you look at the network and console section to make sure nothing is there.

Also to note, I am not a contributor. But I feel like you need to provide some more context to your situation.

**Comment by breakTBB** [comment]
> What additional information can you provide? Is your network connection setup? Launch the console and you look at the network and console section to make sure nothing is there.
> 
> Also to note, I am not a contributor. But I feel like you need to provide some more context to your situation.

the network is fine, but the model's answer is not related to my vault, it seems RAG workflow doesn't work correctly.

**Comment by KyleJFischer** [comment]
You need to provide a bit more information as It works for me personally.

What is your setup? Local ollama or OpenAI? What models are you using? Have you updated the plugin to the latest version?  Have you indexed your notes recently? 

Basically, we need information about how you are using it or else, no one can help you.

**Comment by JulianSchnaars** [comment]
I have the same issue. Fairly new Obsidian install. Ollama setup on my Macbook Pro M3. I tried llama3 and gemma2. Plugin version 2.5.2.

Indexing is set to "On Mode Switch", but I also clicked the button to manually refresh vault indexing.
Console says:
- Set chain: vault_qa
- No vectors found in VectorDB for dochash: [...]

**Comment by JulianSchnaars** [comment]
I did some digging. For me all files were filtered out because of this line:
```
return file.stat.mtime > latestMtime
```
mtime always had the same value as latestMtime. not sure where this issue comes form though...
I changed the filter to always return true for the first run and the indexing worked.
